geometry_msgs/Point origin
int32 region_count
geometry_msgs/Point[] positions
float64[] region_start_degrees
float64[] region_stop_degrees
float64[] start_offsets
